File: rest-service/src/main/java/mb/api/model/IOJobPostResponse.java
package mb.api.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import mb.api.model.io.JsonKeys;

@JsonDeserialize(as = IOJobPostResponseImpl.class)
public interface IOJobPostResponse
{
  @JsonProperty(JsonKeys.JobId)
  String getJobId();

  @JsonProperty(JsonKeys.JobId)
  IOJobPostResponse setJobId(String jobId);
}

File: rest-service/src/main/java/mb/api/model/IOJobPostResponse.kt
package mb.api.model

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
import com.fasterxml.jackson.annotation.JsonProperty
import mb.api.model.io.JsonKeys

@JsonInclude(NON_NULL)
data class IOJobPostResponse(@JsonProperty(JsonKeys.JobID) val jobId: String?)